system/system_monitor/include/system_monitor/hdd_monitor/hdd_monitor.hpp

```diff
@@ -32,13 +33,35 @@
  */
 struct HDDParam
 {
-  std::string device_;  //!< @brief device
-  float temp_warn_;     //!< @brief HDD temperature(DegC) to generate warning
-  float temp_error_;    //!< @brief HDD temperature(DegC) to generate error
-  int free_warn_;       //!< @brief HDD free space(MB) to generate warning
-  int free_error_;      //!< @brief HDD free space(MB) to generate error
+  std::string device_;                      //!< @brief device
+  float temp_warn_;                         //!< @brief HDD temperature(DegC) to generate warning
+  float temp_error_;                        //!< @brief HDD temperature(DegC) to generate error
+  int power_on_hours_warn_;                 //!< @brief HDD power on hours to generate warning
+  int total_data_written_warn_;             //!< @brief HDD total data written to generate warning
```

Comment:
@kk-inoue-esol This should be `uint64_t` because TBW of my SSD is 5256000000000000(5256TB), so the value is overflowed.